generators/client/templates/angularjs/src/main/webapp/app/blocks/interceptor/_notification.interceptor.js

```diff
@@ -16,7 +16,7 @@
 
         function response (response) {
             var headers = Object.keys(response.headers()).filter(function (header) {
-                return header.endsWith('app-alert') || header.endsWith('app-params');
```

Comment:
Very similar line (using `endsWith`) exists at `_alert-error.directive.js`. It could need to be changed as well.